test/test_integration_cluster.rb

```diff
@@ -564,6 +564,19 @@ def test_hook_data
     assert 'index 1 data 1', ary[1]
   end
 
+  def test_after_worker_shutdown_hook
+    cli_server "-C test/config/after_shutdown_hook.rb test/rackup/hello.ru"
+    get_worker_pids 0, 2 # make sure workers are booted
+    stop_server
+
+    ary = Array.new(2) do |_index|
+      wait_for_server_to_match(/(after_worker_shutdown worker=\d status=\d+)/, 1)
+    end.sort
+
+    assert 'after_worker_shutdown worker=0 status=0', ary[0]
+    assert 'after_worker_shutdown worker=1 status=0', ary[1]
```

Comment:
```suggestion
    assert_equal 'after_worker_shutdown worker=0 status=0', ary[0]
    assert_equal 'after_worker_shutdown worker=1 status=0', ary[1]
```

Reply:
Good catch. I was pattern matching on the test above. Should I fix that too?

Reply:
Fixed. I opened a separate PR to fix the `assert` I copied (plus some others) https://github.com/puma/puma/pull/3784